Debug-custom-Pylint-check-tests.md

```diff
@@ -0,0 +1,49 @@
+## Table of contents
+
+* [Introduction](#introduction)
+* [Run a single checker](#run-a-single-checker)
+  * [Print statements](#print-statements)
+  * [The Python debugger](#the-python-debugger)
+
+## Introduction
+
+When trying to debug the tests for a custom Pylint checker, remember that the tests are just like any other backend test. You may find our [[guide to debugging backend tests|Debug-backend-tests]] useful. That said, there are some helpful tips for working with Pylint checkers in particular. We describe them below.
+
+## Run a single checker
+
+Debugging a Pylint checker's tests often comes down to debugging the checker itself. To see how your checker behaves on some code, put that code into `test.py`. Then assuming that your checker has message with symbol `my-checker-message`, you can run your checker on `test.py` like this:
```

Comment:
Perhaps describe what sort of output is expected to be produced by this process, so that it's clear to the reader how they can use that output for debugging.